kdump/analyse-basic/runtest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2019 Red Hat, Inc.
+#
+#   This program is free software: you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License as
+#   published by the Free Software Foundation, either version 2 of
+#   the License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE.  See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program. If not, see http://www.gnu.org/licenses/.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+# Source kdump common functions
+. ../include/runtest.sh
+
+analyse()
+{
+    GetCorePath
+    LsCore
```

Comment:
This is a basic test for now. But later if it's changed to analyse-vmcore.
We have another function to flag and check system status.  like thits.
CheckUnexpectedReboot()
{
    if [ -f "$K_REBOOT" ]; then
        rm -f "$K_REBOOT"
        MajorError "- Unexpected reboot is detected. Please check if system has \
been rebooted from panic or other possible incidents."
    fi
    touch "$K_REBOOT"
    sync;sync;
}